integrations/SnapEngage/browser.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import get from "get-value";
+import logger from "../../utils/logUtil";
+
+// import { NAME } from "./constants";
+import { recordingLiveChatEvents } from "./util";
+import { getHashFromArrayWithDuplicate } from "../utils/commonUtils";
+// import { LOAD_ORIGIN } from "../ScriptLoader";
+
+class SnapEngage {
+  constructor(config) {
+    this.widgetId = config.widgetId;
+    this.recordLiveChatEvents = config.recordLiveChatEvents;
+    this.eventsToStandard = config.eventsToStandard;
+    this.updateEventNames = config.updateEventNames;
+  }
+
+  loadScript() {
+    window._paq = window._paq || [];
+    (function (widgetId) {
+      const se = document.createElement("script");
+      se.type = "text/javascript";
+      se.async = true;
+      se.src = `https://storage.googleapis.com/code.snapengage.com/js/${widgetId}.js`;
+      let done = false;
+      se.onload = se.onreadystatechange = function () {
+        if (
+          !done &&
+          (!this.readyState ||
+            this.readyState === "loaded" ||
+            this.readyState === "complete")
+        ) {
+          done = true;
+          /* Place your SnapEngage JS API code below */
+          /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
+        }
+      };
+      const s = document.getElementsByTagName("script")[0];
+      s.parentNode.insertBefore(se, s);
+    })(this.widgetId);
```

Comment:
We can do this using `scriptLoader()`.

Reply:
We have to update the scriptLoader function to load their SDK. So it is not required.